src/api/student/homeworks.ts

```diff
@@ -0,0 +1,45 @@
+import {type Account, type Session, SessionTokenRequired, ComingHomework} from "~/models";
+import {Request} from "~/core/request";
+import {decodeComingHomework, decodeHomework} from "~/decoders/homework";
+import {Homework} from "~/models";
+
+export const studentHomeworks = async (
+  session: Session,
+  account: Account,
+  date: string
+): Promise<Homework[]> => {
+  if (!session.token)
+    throw new SessionTokenRequired();
+
+  const request = new Request(`/Eleves/${account.id}/cahierdetexte/${date}.awp?verbe=get&v=4.62.1`)
```

Comment:
```suggestion
  const request = new Request(`/Eleves/${account.id}/cahierdetexte/${date}.awp?verbe=get`)
```

La version est déjà ajoutée en dessous avec `.addVersionURL()`